interface/wx/string.h

```diff
@@ -916,6 +916,41 @@ class wxString
     */
     int CmpNoCase(const wxString& s) const;
 
+    /**
```

Comment:
The functions documented here don't see to exist.

Reply:
I'm a little confused. I don't think I made CmpNoCase(). It certainly seems to exist in wxString, and I can't find anything else in this file that's mine.

Reply:
You didn't make `CmpNoCase()`, but you did add (and didn't remove/move) the documentation for `CmpNatural[NoCase]()` here, which shouldn't exist because these methods themselves don't exist.

Reply:
Oops. What confused me was looking in the wrong string.h.  Done.